### ISSUE TITLE
ckb v0.2.6 No devices detected K65LUX 

### ISSUE THREAD

**Issue by Midnightfender** [ccMSC/ckb#471]
I am getting no devices connected when trying to use ckb. Using a K65LUX on macOS(sierra).

**Comment by joonne** [comment]
Same for me using Ubuntu 16.04.

**Comment by horen20wash** [comment]
You have to checkout and switch to the newdev branch and build ckb. The keyboard will work.

**Comment by Midnightfender** [comment]
So Ive compiled the newdev branch and the exact same thing, no devices found, Im on the latest firmware as well.

**Comment by Midnightfender** [comment]
Just for posterity sake I have also compiled and tried every available branch. None detect the device, reboots/disconnect-reconnects included.

Not sure if there was a firmware update. But I am running macOS Sierra as well.

**Comment by horen20wash** [comment]
Every branch does not help, only newdev branch will support LUX keyboards. Did you verify that your device is is in the usb.h of the code you are compiling.

**Comment by Midnightfender** [comment]
Yes, I actually got it working. I was forced to build it and fix a couple issues that popped up in the build debug. After complete build in Qt and demon relocation and linking everything seems functional. If I get a chance Ill be posting the prebuilt w/ demon install direcs.
